### PR TITLE
New EmojiPlugin UX

### PR DESCRIPTION
**Pull request by adriantran**

### CODE REVIEW

File: packages/roosterjs-react-emoji/lib/components/emoji.scss [hunk 1]
```diff
@@ -12,8 +12,56 @@
...
+    left: 0;
+    z-index: 1;
+    border-bottom: solid 1px #dadada;
+    padding: 0 $paneHorizontalPadding;
+
+    button {
+        background: $buttonColor;
+        height: $buttonHeight;
+        width: $buttonWidth;
+        margin-right: 0 !important;
```
Comment (pranavgk) [inline packages/roosterjs-react-emoji/lib/components/emoji.scss]: Can you add a comment about the need for `!important`?

File: packages/roosterjs-react-emoji/lib/plugins/EmojiPlugin.tsx [hunk 2]
```diff
@@ -92,7 +99,7 @@ export default class EmojiPlugin implements EditorPlugin {
         }
     }
 
-    public startEmoji(startingString: string = ":"): void {
+    public startEmoji(startingString: string = ':'): void {
```
Comment (pranavgk) [inline packages/roosterjs-react-emoji/lib/plugins/EmojiPlugin.tsx]: When do you use single quote vs double quote?

File: packages/roosterjs-react-emoji/lib/components/emoji.scss [hunk 3]
```diff
@@ -12,8 +12,56 @@
...
+    overflow-y: auto;
+}
+
+.fullListContentContainer {
+    margin-top: $buttonHeight;
+}
+
+.partialListContent,
+.fullListContent {
+    width: $paneWidth;
```
Comment (pranavgk) [inline packages/roosterjs-react-emoji/lib/components/emoji.scss]: Do you plan to fix the IE11 issue in a separate PR?

Comment (adriantran) [reply packages/roosterjs-react-emoji/lib/components/emoji.scss]: same pr

File: packages/roosterjs-react-emoji/lib/components/emoji.scss (hunk 1 above)
Comment (adriantran) [reply packages/roosterjs-react-emoji/lib/components/emoji.scss]: sure

**Pull request merged**